activities/value-and-impact/user-mapping/index.md

```diff
@@ -31,4 +31,3 @@ Other interested parties:
 
 * end users (residents and citizens)
 * journalists and academia
-
```

Comment:
What is the reason for removing this line at the end of the file?

Reply:
delinting, I believe.